File: nelio-alves-algoritmo-e-logica-de-programacao/c/exercicios/negativos.c
#include <stdio.h>

int main() {
    int N, vet[N];

    printf("Quantos numeros voce vai digitar?");
    scanf("%d", &N);

    for (int i = 0; i < N; i++) {
        printf("Digite um numero:");
        scanf("%d", &vet[i]);
    }

    printf("\nNUMEROS NEGATIVOS:\n");
    for (int i = 0; i < N; i++) {
        if (vet[i] < 0) {
            printf("%d\n", vet[i]);
        }
    }

    return 0;
}

File: nelio-alves-algoritmo-e-logica-de-programacao/python/exercicios/negativos.py
N: int

N = int(input("Quantos numeros voce vai digitar? "))

vet: [int] = [0 for x in range(N)]

for i in range(N):
	vet[i] = int(input("Digite um numero: "))

print("NUMEROS NEGATIVOS:")
for i in range(N):
	if vet[i] < 0:
		print(vet[i])